cmd/snap/cmd_snap_op.go

```diff
@@ -202,18 +203,24 @@ func refreshAll() error {
 	if err != nil {
 		return fmt.Errorf("cannot list updates: %s", err)
 	}
+	// nothing to update/list
+	if len(updates) == 0 {
+		return nil
```

Comment:
We discussed this online and agreed to fix consistency on the other end: fix list so if there's no entry it shows absolutely no output. It's not an error (exit 0), and it shouldn't show even the header unless there's at least one entry.